instagram-app/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import dummyData from './dummy-data';
+import PostContainer from './components/PostContainer';
+
+import './App.css';
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      data: dummyData
+    };
```

Comment:
Quick little tip, when using `create-react-app` you no longer need to wrap your state in a `constructor` and call `super()` 

So you can take this
```
 constructor() {
    super();
    this.state = {
      data: dummyData
    };
  }
```

and convert it to this
```
state = {
  data: dummyData
};
```

Right off the bat you can see this is already much more concise and easier to read. The reason you can do this is cause of tool named [babel](https://babeljs.io/). Babel is a transpiler, which means it takes the current code you've written and rewrites it into the same language typically just to a different syntax/version that's well supported on different browsers/environments. This allows for use on super old browsers like IE

for example if you write
```
const foo = () => {
  return 'bar'
}
```

babel with transpiles that into
```
var foo = function foo() {
    return 'bar';
}
```

you can see it has the exact same functionality but just a different syntax that is well supported on all web browsers



So remember as long as your react app is bootstrapped with `create-react-app` when your state isn't wrapped in a constructor and calling super babel automatically adds that code in there so you don't have to.